global.json

```diff
@@ -1,5 +1,6 @@
 {
   "sdk": {
-    "version": "5.0.203"
+    "version": "5.0.203",
```

Comment:
I needed to change something in the global.json file.

Unfortunally, you only can download SDK Version 5.0.202 or 5.0.3xx from the official .NET Website.
Because my SDKs MUST match 5.0.203 (which you cannot download (maybe I only didn't found it) I use SDK 5.0.202 (which is lower) and SDK 5.0.3xx (which is the next feature version).

I do recommend to set version to 5.0.202 (which you still can download) and use an own rollForward (for example latestMajor/Minor (in the future when new features are added) (and on my machine was per default only installed the latest (SDK) version).

Reply:
I found the SDK 5.0.203 after a little deeper search.